CRITICAL - 2023-05-22 00:13:30 --> The namespace delimiter is a backslash (\), not a slash (/). Route handler: "\App\Controllers\Users/UsersController::index"
in SYSTEMPATH\Router\Router.php on line 474.
 1 SYSTEMPATH\Router\Router.php(474): CodeIgniter\Router\Exceptions\RouterException::forInvalidControllerName('\\App\\Controllers\\Users/UsersController::index')
 2 SYSTEMPATH\Router\Router.php(182): CodeIgniter\Router\Router->checkRoutes('users')
 3 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('users')
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-22 00:13:42 --> The namespace delimiter is a backslash (\), not a slash (/). Route handler: "\App\Controllers\Users/UsersController::index"
in SYSTEMPATH\Router\Router.php on line 474.
 1 SYSTEMPATH\Router\Router.php(474): CodeIgniter\Router\Exceptions\RouterException::forInvalidControllerName('\\App\\Controllers\\Users/UsersController::index')
 2 SYSTEMPATH\Router\Router.php(182): CodeIgniter\Router\Router->checkRoutes('users')
 3 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('users')
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-22 00:16:13 --> Invalid file: "Users/UsersCreateForm.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('Users/UsersCreateForm.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('Users/UsersCreateForm', [], true)
 3 APPPATH\Controllers\Users\UsersController.php(10): view('Users/UsersCreateForm')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Users\UsersController->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users\UsersController))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-22 00:16:18 --> Invalid file: "Users\UsersCreateForm.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('Users\\UsersCreateForm.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('Users\\UsersCreateForm', [], true)
 3 APPPATH\Controllers\Users\UsersController.php(10): view('Users\\UsersCreateForm')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Users\UsersController->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users\UsersController))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-22 00:16:18 --> Invalid file: "Users\UsersCreateForm.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('Users\\UsersCreateForm.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('Users\\UsersCreateForm', [], true)
 3 APPPATH\Controllers\Users\UsersController.php(10): view('Users\\UsersCreateForm')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Users\UsersController->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users\UsersController))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-22 00:32:50 --> Invalid file: "success_user_form.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('success_user_form.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('success_user_form', [], true)
 3 APPPATH\Controllers\UsersController.php(14): view('success_user_form')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UsersController->create()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-22 00:33:03 --> Invalid file: "success_user_form.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('success_user_form.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('success_user_form', [], true)
 3 APPPATH\Controllers\UsersController.php(14): view('success_user_form')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UsersController->create()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-22 00:34:13 --> Invalid file: "success_user_form.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('success_user_form.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('success_user_form', [], true)
 3 APPPATH\Controllers\UsersController.php(15): view('success_user_form')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UsersController->create()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-22 00:51:09 --> You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 883.
 1 SYSTEMPATH\Model.php(642): CodeIgniter\Database\BaseConnection->table(null)
 2 SYSTEMPATH\Model.php(234): CodeIgniter\Model->builder()
 3 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UsersController->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-22 00:57:11 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\app\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\UsersController->index()
#8 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
#9 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\app\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\UsersController->index()
#7 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
#8 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-05-22 00:57:11 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UsersController->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-22 00:58:00 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\app\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\UsersController->index()
#8 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
#9 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\app\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\UsersController->index()
#7 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
#8 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-05-22 00:58:00 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UsersController->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-22 00:58:25 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\app\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\UsersController->index()
#8 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
#9 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\app\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\UsersController->index()
#7 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
#8 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-05-22 00:58:25 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UsersController->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-22 00:59:34 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\app\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\UsersController->index()
#8 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
#9 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\app\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\UsersController->index()
#7 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
#8 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-05-22 00:59:34 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UsersController->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-22 00:59:41 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\app\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\UsersController->index()
#8 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
#9 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\app\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\UsersController->index()
#7 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
#8 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-05-22 00:59:41 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UsersController->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-22 01:00:00 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\app\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\UsersController->index()
#8 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
#9 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\app\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\UsersController->index()
#7 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
#8 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-05-22 01:00:00 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UsersController->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-22 01:00:07 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\app\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\UsersController->index()
#8 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
#9 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\app\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\UsersController->index()
#7 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
#8 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-05-22 01:00:07 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UsersController->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-22 01:02:58 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\app\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\UsersController->index()
#8 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
#9 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\app\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\UsersController->index()
#7 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
#8 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\actividades\4-3-desarrollo-aplicacion-con-mvc\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-05-22 01:02:58 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\UsersController.php(15): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UsersController->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsersController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
